TROUBLESHOOTING.md

```diff
@@ -0,0 +1,82 @@
+# Troubleshooting BlueALSA
+
+Solutions to some of the most commonly encountered errors when using BlueALSA.
+
+## Using BlueALSA alongside PulseAudio or PipeWire
+
+Due to BlueZ limitations, only one program can register as provider of
```

Comment:
I think, that maybe it should be worded differently. When I was writing this, my knowledge about BlueZ was rather limited :D

Firstly, it's not BlueZ limitation, but a design. Secondly, only one program **should** register if one would like to have deterministic behavior. And thirdly, for A2DP it's not true that it's not possible to run BlueALSA with PulseAudio/PipeWire (with their BT modules enabled). The case is that in such setup first Bluetooth device will be connected with an application which has registered itself before other apps.

So, maybe something like this would be better:

> It is not advised to run BlueALSA if either PulseAudio or PipeWire are also running with their own
Bluetooth modules enabled. If one would like to have a deterministic setup, it is first necessary to disable Bluetooth in those applications.